src/05-derivatives/05-diffusion-derivatives.md

```diff
@@ -0,0 +1,381 @@
+# Diffusion derivatives
+
+## Preprocessed diffusion weighted images
+
+Multiple different versions of preprocessing can be stored for the same source
+data. To distinguish them from each other, the `desc` fileneame keyword can be
+used. Details of preprocessing performed for each variation of the processing
+should be included in the pipeline documentation.
+
+```Text
+<pipeline_name>/
+    sub-<participant_label>/
+        dwi/
+          <source_keywords>[_referencemap-<referencemap>][_desc-<label>]_dwi.nii[.gz]
+          <source_keywords>[_referencemap-<referencemap>][_desc-<label>]_dwi.bvals
+          <source_keywords>[_referencemap-<referencemap>][_desc-<label>]_dwi.bvecs
+```
+
+The JSON sidecar file is REQUIRED (due to the REQUIRED `SkullStripped` field -
+see [Common Data Types](02-common-data-types.md)), and if present can be used to
+store information about what preprocessing options were used (for example
+whether motion compensation was performed, non-linear corrections were applied,
+whether Eddy current correction was performed, denoising, or intensity
+normalization were applied, etc).
+
+Additional reserved JSON metadata fields:
+
+| **Key name**           | **Description**                                                                                                          |
+| ---------------------- | ------------------------------------------------------------------------------------------------------------------------ |
+| MotionCompensation     | OPTONAL. Boolean. Motion compensation                                                                                    |

```

Comment:
Should this be "MotionCorrection" instead of "MotionCompensation"? 

Should we store a tsv with motion correction parameters, similar to what is proposed above for the fMRI derivatives? I don't think it's different in principle.